atc/worker/worker.go

```diff
@@ -609,9 +611,13 @@ func (worker *gardenWorker) cloneRemoteVolumes(
 			"dest-worker": inputVolume.WorkerName(),
 		}
 
+
+		maxInFlight <- 1
```

Comment:
Hey, should this be here? I wonder if it should instead be right in the beginning of the `for` loop 🤔  

It seems to me that having it here, it feels like we'd end up with, e.g., 6 volumes created, while only 5 streaming ops go on (leaving that volume there wait for its turn). 

Although that might not be a big of a deal, it seems to me that "reserving access" right in the beginning would make it more "even" (5 volumes, while 5 ops go on).

(unless that was really the desire - having a "pre-created" volume there just waiting for its turn to get into the streaming phase - which I think seems like a valid reasoning).

thanks!!